src/web/components/CommentsLayout.tsx

```diff
@@ -56,49 +61,123 @@ export const CommentsLayout = ({
     discussionApiClientHeader,
     commentToScrollTo,
     onPermalinkClick,
-}: Props) => (
-    <Flex direction="row">
-        <LeftColumn showRightBorder={false}>
-            <SignedInAs
-                pillar={pillar}
-                enableDiscussionSwitch={enableDiscussionSwitch}
-                user={user}
-                commentCount={commentCount}
-                isClosedForComments={isClosedForComments}
-            />
-        </LeftColumn>
-        <div className={containerStyles}>
-            <Hide when="above" breakpoint="leftCol">
-                <div className={bottomPadding}>
-                    <SignedInAs
-                        pillar={pillar}
-                        enableDiscussionSwitch={enableDiscussionSwitch}
+}: Props) => {
+    const [heightToStick, setHeightToStick] = useState(DEFAULT_HEIGHT);
+    const [heightChangeCount, setheightChangeCount] = useState<number>(0);
+    const heightRef = useRef<HTMLDivElement>(null);
+
+    const handleHeightChange = () => {
+        setheightChangeCount(heightChangeCount + 1);
+    };
+
+    const checkHeight = (ref: RefObject<HTMLDivElement>) => {
+        if (ref.current) {
+            const availableHeight = ref.current.clientHeight;
+            // Don't run the sticky container all the way to the bottom
+            let heightWereUsing = availableHeight - 600;
```

Comment:
Nitpick: `heightWereUsing` vs `heightWeAreUsing`, because you can't have apostrophes in var names.

Reply:
Thanks. This bugged me too but I never got around to fixing it